spec/features/cart/checkout_spec.rb

```diff
@@ -19,6 +19,18 @@
     end
 
     it 'Theres a link to checkout' do
+      user = User.create!(name: "Steve",
+                          address:"123 Main St.",
```

Comment:
in a lot of these old tests they did not include a user to create orders/checkout

Reply:
same with my user(role: 1) were created in tests without a merchant. dashboard was breaking those. fixed now.